scoped/authentication/login-indirect/login-indirect.md

```diff
@@ -0,0 +1,32 @@
+**Name**
+Login (indirect)
+
+----
+
+**Actor**  
+Anonymous user
+
+----
+ 
+**Subject area**  
+Login page or modal on any page
+
+----
+
+**Preconditions**  
+Not authenticated, the user has tried to perform an authenticated action, it is unknown if the user is existing or new
+
+----
+
+**Termination outcome**
+User is signed in and back on the original page or home page, or routed to sign up process
+
+----
+
+**Basic flow**  
+
+- User presented with email and password fields and login button
+- User presented with various prompts under the fields based on action
+- Any errors are surfaced in the same manner, errors are clearly identified using red
+- Form will not submit unless all fields are filled
+- Form cannot resubmit if user has already submitted form
```

Comment:
Can't comment on scoped/authentication/login-indirect/login-indirect.png so commenting here.  Is "want to answer?" in the diagram contextual?  (e.g. would it be "want to post?" or "want to vote?" etc for other actions?)  We don't want to show "want to answer?" if the person tried to ask a question; either using generic language reflecting the specific action that was attempted is fine.

Reply:
This would be contextual to the site actually. Answer could be anything. Perhaps wrapping in `${}` might denote this better?

Reply:
I'm not sure what you mean by "to the site"; it seems more contextual to the *action*, like what button/link the person clicked on before getting told to log in.  On a question page this would be "log in to answer", which was the use case that led to this I think, but the same issue would arise of someone who's not logged in clicks on "ask question".  So either that top line needs to vary based on how the user came to be facing that dialogue, or we need to do something generic.

Reply:
sorry im referring to the instance - so writing.codidact.com vs programming.codidact.com might use different terminology or is there a way for us to just use a more generic term here like post in your example

Reply:
Let's just do that -- instead of "answer" say "post", and that will cover us for a while.  We can revisit when we have other use cases.  I'm not trying to make this complicated. :-)